### PR TITLE
Split functions to L10n

### PR DESCRIPTION
**Pull request by zeroadam**
split functions from text.php to L10n class.
Related to: #3878

### CODE REVIEW

File: src/Core/L10n.php [hunk 1]
```diff
@@ -345,4 +346,67 @@ public static function getAvailableLanguages()
...
+	 * @param string $s String with day or month name.
+	 * @return string Translated string.
+	 */
+	public static function getDay($s)
+	{
+		$ret = str_replace(['Monday','Tuesday','Wednesday','Thursday','Friday','Saturday','Sunday'],
+			[self::t('Monday'), self::t('Tuesday'), self::t('Wednesday'), self::t('Thursday'), self::t('Friday'), self::t('Saturday'), self::t('Sunday')],
+			$s);
+
+		$ret = str_replace(['January','February','March','April','May','June','July','August','September','October','November','December'],
```
Comment (annando) [inline src/Core/L10n.php]: Can you add spaces before the commas?

**Comment by annando** [comment]
On this occasion I have to ask: Do you run your instance with your changed code? I'm just asking, since the last PR generated a WSOD at a very central place - which normally should be avoidable when running it for some hours on the local machine.

**Comment by annando** [comment]
Okay, that's great.

**Comment by zeroadam** [comment]
@annando I think I missed the `use Feature` because I didn't do any searching, just clicking around the interface and such. I'll try to be more thorough going forward!

**Comment by zeroadam** [comment]
@MrPetovan haha yes.. I believe there was only one that needed to be added!

**Pull request merged**